flixel/util/FlxPath.hx

```diff
@@ -207,12 +191,21 @@ class FlxPath implements IFlxDestroyable
 	
 	/**
 	 * Internal function for moving the object along the path.
-	 * Generally this function is called automatically by preUpdate().
 	 * The first half of the function decides if the object can advance to the next node in the path,
 	 * while the second half handles actually picking a velocity toward the next node.
 	 */
 	public function update(elapsed:Float):Void
 	{
+		if (object == null)
+			return;
+		
+		if (_firstUpdate)
```

Comment:
Hm, I'm not sure that approach is better. It leads to an even tigther coupling between `FlxPath` and `FlxObject` (accessing `path._desiredImmovable `), I already don't like it as is..